tests/integration/test_methodologies_para_ai_rules_heuristics.py

```
@@ -1314,7 +1314,12 @@ def test_evaluate_ollama_not_available(self, tmp_path: Path) -> None:
 
         f = _make_file(tmp_path, "doc.txt")
         h = AIHeuristic()
-        with patch.object(_mod, "OLLAMA_AVAILABLE", False):
+        # Patch both OLLAMA_AVAILABLE and ollama to prevent any real or leaked
+        # mock client from reaching _parse_response (xdist isolation defence).
+        with (
+            patch.object(_mod, "OLLAMA_AVAILABLE", False),
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)